core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -156,6 +154,13 @@ object Defaults {
   val OffsetCommitTimeoutMs = OffsetConfig.DefaultOffsetCommitTimeoutMs
   val OffsetCommitRequiredAcks = OffsetConfig.DefaultOffsetCommitRequiredAcks
 
+  /** ********* Transaction management configuration ***********/
+  val TransactionsTopicMinISR = TransactionLog.DefaultMinInSyncReplicas
+  val TransactionsLoadBufferSize = TransactionLog.DefaultLoadBufferSize
+  val TransactionsTopicReplicationFactor = TransactionLog.DefaultReplicationFactor
+  val TransactionsTopicPartitions = TransactionLog.DefaultNumPartitions
+  val TransactionsTopicSegmentBytes = TransactionLog.DefaultSegmentBytes
+
```

Comment:
Sholud we also have a config for unclean leader election being default off? I think that was a suggestion on the doc (though I am not sure if the doc was updated).